platform/linux-generic/pktio/socket_xdp.c

```diff
@@ -291,6 +313,62 @@ static int sock_xdp_close(pktio_entry_t *pktio_entry)
 	return 0;
 }
 
+static int sock_xdp_stats(pktio_entry_t *pktio_entry, odp_pktio_stats_t *stats)
+{
+	memcpy(stats, &pktio_entry->s.stats, sizeof(odp_pktio_stats_t));
```

Comment:
A lock is likely required to avoid race with RX/TX functions.

Reply:
Locking added in v2.